config/environments/production.rb

```diff
@@ -94,4 +94,19 @@
 
   # Serve assets under public
   # config.serve_static_assets = true
+
+  config.action_mailer.default_url_options = { :host => 'gthc.io' }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = true
+
+  ActionMailer::Base.smtp_settings = {
+    :user_name => Rails.application.secrets[:gthc_email],
```

Comment:
Like I mentioned in my other comment, let's use environment variables, let's stay consistent with other non-public values, and we can integrate confidentials across the board once we go 5.2.

Reply:
Ok I'll hold of this PR until we have 5.2 on staging

Reply:
No build it with ENV for now, and we can have a whole separate branch for moving to credentials if we sit it fit our needs better. Parts of this PR is also necessary on admin as it also uses Devise and has a forgot password option.

Reply:
@mafuvadze

Reply:
@mafuvadze Any update my friend??

Reply:
Can you please update this?

Reply:
@mafuvadze updated